spot_driver/src/api/default_image_client.cpp

```diff
@@ -197,6 +126,22 @@ tl::expected<std::vector<geometry_msgs::msg::TransformStamped>, std::string> get
   }
   return out;
 }
+
+tl::expected<sensor_msgs::msg::CompressedImage, std::string> toCompressedImageMsg(
+    const bosdyn::api::ImageCapture& image_capture, const std::string& robot_name,
+    const google::protobuf::Duration& clock_skew) {
+  const auto& image = image_capture.image();
+  if (image.format() != bosdyn::api::Image_Format_FORMAT_JPEG) {
+    return tl::make_unexpected("Only JPEG image cannot be sent as ROS2-compressed image.");
```

Comment:
Having an issue while testing on an RGB spot where I constantly get this error and no images (camera info, depth, compressed, or rgb) get published. Looking into how to resolve this